datastore/google/cloud/datastore/_onestore_v3.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto2";
```

Comment:
@pcostell Thanks for the pointer (I wrote this by hand based on inspecting the raw bytes from a reference)